src/components/KPagination/KPagination.vue

```diff
@@ -1,16 +1,21 @@
 <template>
   <nav
+    ref="kPaginationElement"
     class="k-pagination"
+    :class="{ 'page-jump': !disablePageJump && !offset }"
     data-testid="k-pagination"
   >
     <template v-if="!offset">
+      <!-- hidden on mobile (below kui-breakpoint-mobile) -->
```

Comment:
let's remove the HTML comments so they don't render in the DOM and place them in the script area by the variables if needed (both instances)

Reply:
Done